README.md

```diff
@@ -84,15 +84,22 @@ After **the issue** is resolved and merged, `hotfix` branch will be deleted and
 ## Github, Gitlab(s), etc?
 
 We use github for common activities, as it is currently most easily discoverable space.
-**UNIGE Platform is deployed with CI/CD from integral gitlab**, which is pushing changes with github.
-It is also used for internal astro-only projects.
+Hence github hosts the **"community version"** of the software.
+
+**UNIGE MMODA Site is deployed with CI/CD from integral gitlab**, which is pushing changes with github.
+It is also used for internal astro-only projects (like product gallery).
+Much of the frontend also has the reference **"author's version"** in gitlab, managed by the frontend developer alone.
 
 **where to push?** 
-* Any change to gitlab.astro.unige.ch is synchronized to github.
-* not every change to github is synchronized to gitlab.
+  * considering that:
+    * Any change to gitlab.astro.unige.ch is expected to be synchronized to github, but it's not necessarily done automatically, since it may require merging.
```

Comment:
In which cases is the merging required? I though that any push on the gitlab repo is automatically mirrored towards the github repo.

Reply:
I just removed it, as explained in the other comment. Any change in github is destroyed by the mirroring, even new branches. This prevents any kind of community-version idea.